data/pki/meson.build

```diff
@@ -1,7 +1,7 @@
 # only install files that are going to be used
-if libjcat.version().version_compare ('>= 0.1.9')
-  supported_gpg = libjcat.get_pkgconfig_variable('supported_gpg')
-  supported_pkcs7 = libjcat.get_pkgconfig_variable('supported_pkcs7')
+if meson.version().version_compare('>=0.51.0')
```

Comment:
shouldn't it be both meson and libjcat check?

Reply:
Based on discussion in the #mesonbuild IRC channel, I recommended the use of default_value to do fallback in the event that the dependency is too old to support this variable. I assume that's probably the real reason the dependency check was being done in the first place.

And, excessive nesting was a concern so separately checking the libjcat version and the meson version and doing a four-way branch is probably not wanted. (This is also the reason to not support exactly meson 0.50.0 via fallback to get_pkgconfig_variable which only works when not a subproject.)